assets/dist/rolodex.js

```diff
@@ -1,5 +1,5 @@
 (function() {
```

Comment:
we haven't run `gulp build` since merging PRs from last week. i wrote new gulp tasks to do it better, so I checked this in.

ideally, we only create this file when we tag a new rolodex release